nodejs/tfwconnector/MessageSender.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+let TFWConnector = require('./TWFConnector');
+
+
+class MessageSender{
+
+    constructor(connector){
```

Comment:
`connector` should be optional, and if the user does not provide one it should be initialised.